AISKU/Tests/Unit/src/ApplicationInsightsDeprecatedTests.ts

```diff
@@ -19,7 +19,11 @@ export class ApplicationInsightsDeprecatedTests extends AITestClass {
     
     constructor() {
         super("ApplicationInsightsDeprecatedTests");
+
+        // this.assertNoEvents = false;
```

Comment:
This is a left over from before fixing all cleanup paths -- probably should cleanup before release (proposing not this PR)